include/beast/http/write.hpp

```diff
@@ -17,6 +17,120 @@
 namespace beast {
 namespace http {
 
```

Comment:
Style consideration [member grouping](http://www.stack.nl/~dimitri/doxygen/manual/grouping.html#memgroup) to avoid copy-paste of the common parameters in these functions.